src/quo/components/list_items/account/view.cljs

```diff
@@ -11,8 +11,7 @@
     [reagent.core :as reagent]))
 
 (defn- account-view
-  [{:keys [account-props title-icon? blur? theme]
-    :or   {title-icon? false}}]
+  [{:keys [account-props title-icon blur? theme]}]
```

Comment:
nice! also just an observation about the removed code I thought I would share here.
In general we should have booleans as false by default. i.e they are something you have to activate not disable 👍

Reply:
True that!

--

Adding context on why the boolean prop was replaced:

In the old code, we had `title-icon?` a boolean. If it's true, the component displays the keycard icon. 

We need the flexibility to display the icon based on the context. So, this PR replaces `title-icon?` with `title-icon` that receives the icon name and displays.